src/main/java/seedu/address/commons/util/XmlUtil.java

```diff
@@ -68,4 +73,41 @@
         m.marshal(data, file.toFile());
     }
 
+    /**
+     * Update the user with the {@code currentAccount} to the new password.
+     * @param file The valid xml file containing the account list stored.
+     * @param currentAccount The associated Account of which the password is to be changed.
+     * @param newPassword The new password to change to
+     * @throws IllegalValueException Thrown if duplicated account was found in the account list.
+     * @throws JAXBException Thrown if there is an error during converting the data
+     * into xml and writing to the file.
+     * @throws FileNotFoundException Thrown if the account list file cannot be found.
+     */
+    @SuppressWarnings("unchecked")
```

Comment:
Do confirm if this is actually required!

Reply:
Removed